src/create-zipl-disk-image.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+image="$1"
+
+# supermin appliance
+tmpd=$(mktemp -d) && trap 'rm -rf ${tmpd}' EXIT
+supermin --prepare --use-installed -o "${tmpd}/supermin.d" s390utils-core
+supermin --build --size 1G --format ext2 "${tmpd}/supermin.d" -o "${tmpd}/appliance.d"
+mv "${tmpd}/appliance.d/root" "${image}"
+
+# Shrinking
+e2fsck -f -y "${image}"
+resize2fs -M "${image}"
```

Comment:
is this needed? what's the actual disk usage (`du -sh file.img`) without doing this?

Reply:
It's a sparse file: `304M    /tmp/zipl`